src/main/kotlin/no/nav/syfo/App.kt

```diff
@@ -18,14 +24,31 @@ fun main() {
     val applicationState = ApplicationState()
     val environment = Environment()
     val logger = LoggerFactory.getLogger("ktor.application")
-    val wellKnownInternalAzureAD =
-        getWellKnown(
-            wellKnownUrl = environment.azure.appWellKnownUrl
-        )
-    val azureAdClient =
-        AzureAdClient(
-            azureEnvironment = environment.azure
+
+    val redisConfig = environment.redisConfig
+    val cache = RedisStore(
```

Comment:
Kan ikke se at vi har satt opp redis for denne appen (i naiserator-filene eller egen redis-config.yaml), så dette vil ikke fungere. Men trenger vi egentlig caching her? Hvis vi skulle trenge det så bør det først legges på i egen PR før vi tar det i bruk i en client.